sklearn/feature_extraction/text.py

```diff
@@ -32,7 +32,7 @@
 from ..utils import deprecated
 from ..utils.fixes import frombuffer_empty
 from ..utils.validation import check_is_fitted
-
+from ..utils.fixes import bincount
```

Comment:
I would prefer if you added this import to the one importing 'frombuffer_empty'.